BLUESPAWN-client/src/hunt/hunts/HuntT1122.cpp

```diff
@@ -0,0 +1,70 @@
+#include "hunt/hunts/HuntT1122.h"
+#include "hunt/RegistryHunt.h"
+
+#include "util/configurations/Registry.h"
+#include "util/filesystem/Filesystem.h"
+#include "util/log/Log.h"
+#include "util/log/HuntLogMessage.h"
+
+#include "common/Utils.h"
+
+#include <algorithm>
+
+using namespace Registry;
+
+namespace Hunts{
+
+	HuntT1122::HuntT1122() : Hunt(L"T1122 - COM Hijacking"){
+		dwSupportedScans = (DWORD) Aggressiveness::Intensive;
+		dwCategoriesAffected = (DWORD) Category::Configurations;
+		dwSourcesInvolved = (DWORD) DataSource::Registry;
+		dwTacticsUsed = (DWORD) Tactic::Persistence;
+	}
+
+	int HuntT1122::ScanIntensive(const Scope& scope, Reaction reaction){
+		LOG_INFO("Hunting for " << name << " at level Cursory");
+		reaction.BeginHunt(GET_INFO());
+
+		int detections = 0;
+
+		std::map<std::wstring, std::vector<RegistryKey>> files{};
+
+		for(auto key : CheckSubkeys(HKEY_LOCAL_MACHINE, L"SOFTWARE\\Classes\\CLSID", true, true)){
+			RegistryKey subkey{ key, L"InprocServer32" };
```

Comment:
You need to also check for LocalServer32s

Reply:
and potentially InprocHandler32 as well

Reply:
https://docs.microsoft.com/en-us/windows/win32/com/clsid-key-hklm
Apparently not just those

Reply:
yeah but those are the only file ones for 32bit+. Add the 2 I mentioned then you should be good